pkg/ovs/openflow/ofctrl_flow.go

```diff
@@ -89,6 +90,10 @@ func (f *ofFlow) MatchString() string {
 	return repr
 }
 
+func (f *ofFlow) FlowPriority() string {
```

Comment:
I prefer to return "uint16"

Reply:
The reason why it returned string is that in the indexer where we filter conjuctions with priority, the indexer func expects a []string as return. However I agree it makes more sense to do the string conversion in the indexer function itself. Refactored this.